templates/drone/4/docker-compose.yml.tpl

```diff
@@ -0,0 +1,120 @@
+version: '2'
+services:
+  agent:
+    image: ${drone_agent_image}
+    environment:
+      DRONE_SERVER: ${drone_server}
+      DRONE_SECRET: ${drone_secret}
+{{- if (.Values.http_proxy)}}
+      HTTP_PROXY: ${http_proxy}
+      http_proxy: ${http_proxy}
+{{- end}}
+{{- if (.Values.https_proxy)}}
+      HTTPS_PROXY: ${https_proxy}
+      https_proxy: ${https_proxy}
+{{- end}}
+{{- if (.Values.no_proxy)}}
+      NO_PROXY: ${no_proxy}
+      no_proxy: ${no_proxy}
+{{- end}}
+    volumes:
+      - /var/run/docker.sock:/var/run/docker.sock
+    links:
+      - server:drone
+    command:
+      - agent
+    labels:
+      io.rancher.scheduler.affinity:container_label_soft_ne: io.rancher.stack_service.name=$${stack_name}/$${service_name}
+      io.rancher.container.hostname_override: container_name
+  server:
+    image: ${drone_server_image}
+    environment:
+      DRONE_HOST: drone
```

Comment:
This should be set to your drone server URL, for example `DRONE_HOST=http://drone.mycompany.com`. If not properly configured some features may not work as expected, such as lets encrypt, github commit status, and oauth redirect.

I would therefore not recommend using a dummy value here